haskell/ghc_bindist.bzl

```
@@ -190,3 +190,24 @@ def ghc_bindist(name, version, target):
         target = target,
     )
     native.register_toolchains("@{}//:toolchain".format(toolchain_name))
+
+def haskell_register_ghc_bindists(version):
+    """Register all GHC binaries distributions as toolchains.
```

(minor)

```suggestion
    """Register GHC binary distributions for all platforms/OSs as toolchains.
```
"all" isn't very clear